tests/lib/network_helper.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+from netaddr import *
+import re
+import ipaddress
+def mac_to_snmp_format(mac_addr):
+    if mac_addr != None:
+        try:
+            mac_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",mac_addr)
+            mac = EUI(mac_tmp,dialect=mac_unix)
+            mac_final = str(mac).upper()
+            return mac_final
+        except:
+            assert False , 'Mac Type is Error!'
+    else:
+        assert False , 'Error, MAC is None!'
+
+def ipv4_to_snmp_format(ipv4_str):
+    if ipv4_str != None:
+        ipv4_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",ipv4_str)
+        ipv4_decimal = int(ipv4_tmp,16)
+        ipv4_format = ipaddress.IPv4Address(ipv4_decimal)
+        try:
+            ipv4_address = ipaddress.ip_address(ipv4_format)
+            return ipv4_address
+        except:
+            assert False , 'IPv4 Address Type is Error!'
+    else:
+        assert False , 'Error, IPv4 Address is None!'
+
+def ipv6_to_snmp_format(ipv6_str):
+    ipv6_tmp = re.sub("[\s+\.\!\-\:\/_,$%^*(+\"\')]+|[+()?【】“”！，。？、~@#￥%……&*（）]+',","",ipv6_str)
+    if len(ipv6_tmp) == 32:
+        pattern = re.compile('.{4}')
+        ipv6_tmp_ip = ':'.join(pattern.findall(ipv6_tmp))
+        try:
+            ipv6_address = ipaddress.ip_address(u'%s' %ipv6_tmp_ip)
+            return ipv6_address
+        except:
+            assert False , 'IPv6 Address Type is Error!'
+    else:
+        assert False , 'IPv6 Address Type is Error!'
```

Comment:
Same comments as above, this is quite complex for what it's doing... these two line functions do the same thing:

https://review.gerrithub.io/c/lgirdk/boardfarm/+/448492/3/tests/lib/network_helper.py#14